upstream-fedora-pipeline/AllPackagePipelineStageTrigger

```diff
@@ -101,6 +101,7 @@ pipeline {
                     changeset "config/Dockerfiles/rpmbuild/**"
                     changeset "config/Dockerfiles/cloud-image-compose/**"
                     changeset "config/Dockerfiles/singlehost-test/**"
+                    changeset "config/s2i/jenkins/slave/**"
```

Comment:
Sorry, I forgot until now, but something like this
https://github.com/CentOS-PaaS-SIG/ci-pipeline/blob/master/upstream-fedora-pipeline/AllPackagePipelineStageTrigger#L123..L134
for the slave container will also need to be added (so the tagMap is updated with the new tag and that is passed to build job:)